Duo256m/FreeRTOS/README.md

```diff
@@ -0,0 +1,100 @@
+# BuildRoot Milk-V Duo 256M 测试报告
+
+## 测试环境
+
+### 操作系统信息
+
+- 构建系统：Ubuntu 22.04.3 LTS x86_64
+- 系统版本：Duo-V1.0.9
```

Comment:
Should be 1.10 now?

Reply:
Well... Then I delete these as adding them after test makes more sense?